backend/models/workspaceDocument.js

```diff
@@ -131,13 +131,28 @@ const WorkspaceDocument = {
       workspace: await OrganizationWorkspace.get(`id = ${result.workspace_id}`),
     };
   },
-  where: async function (clause = null, limit = null, withReferences = false) {
+  count: async function (clause = null) {
+    const db = await this.db();
+    const result = await db.get(
+      `SELECT COUNT(*) as total FROM ${this.tablename} ${
+        clause ? `WHERE ${clause}` : ""
+      }`
+    );
+    await db.close();
+    return result.total;
+  },
```

Comment:
is it necessary to return `Count(*) as total` when the default count return value is a single number anyway? Other counts work this way without issue was there an issue?

```
const db = await this.db();
    const result = await db.get(
      `SELECT COUNT(*) FROM ${this.tablename} ${
        clause ? `WHERE ${clause}` : ""
      }`
    );
    await db.close();
    return result;
    ```